browser/src/control/Control.BackstageView.ts

```diff
@@ -1075,6 +1085,14 @@ class BackstageView extends window.L.Class {
 		}
 	}
 
+	private executeAbout(): void {
+		if (this.isStarterMode) {
```

Comment:
I don’t understand why this condition is needed here. 

Both cases call the same function, so we probably don’t need the isStarterMode check.